### PR TITLE
Release v1.0.5

### PR DESCRIPTION
**Pull request by EdwinGuzman**
### Adds

- Exports `ColorCard`.

### Updates

- Updates the `Icon` component to include variants for `actionCheckCircleFilled`, `actionRegistration`, `decorativeEnvelope`, `decorativeLibraryCard` and `decorativeShoppingBag`.
- Updates the docs for the `Colors` Style Guide to include additional data related to contrast and WCAG compliance.
- Updates the `Checkbox` and `Radio` components to use `ui.gray.dark` for the default border color.

### Fixes

- Fixes `useNYPLBreakpoints` hook so it internally controls its own state. This eliminates server/client rendering issues on its initial load.

### CODE REVIEW

File: src/components/StyleGuide/ColorCard.tsx [hunk 1]
```diff
@@ -1,43 +1,178 @@
...
+   * with the current color. */
+  dataWhite: string[];
+  /** The color to use for text in the color card. */
+  textColor: string;
 }
 
-/**
- * ColorCard
- * Component only used for Storybook.
- */
```
Comment (oliviawongnyc) [inline src/components/StyleGuide/ColorCard.tsx:16]: Should we add the above comment back in?

File: src/components/StyleGuide/Colors.stories.mdx [hunk 2]
```diff
@@ -6,82 +6,387 @@ import SimpleGrid from "../Grid/SimpleGrid";
 import { getCategory } from "../../utils/componentCategories";
 import DSProvider from "../../theme/provider";
 
+export const colorContrastData = {
+  brand: {
+    primary: {
+      dataBlack: ["3.45", "Fail", "AA"],
```
Comment (oliviawongnyc) [inline src/components/StyleGuide/Colors.stories.mdx:12]: Just wondering – is there a formula to derive this information, or is hardcoding it the best option?

**Review by oliviawongnyc — APPROVED** [review]

File: src/components/StyleGuide/ColorCard.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/StyleGuide/ColorCard.tsx:16]: Oh, now it's exported for use in Turbine!

File: src/components/StyleGuide/Colors.stories.mdx (hunk 2 above)
Comment (EdwinGuzman) [reply src/components/StyleGuide/Colors.stories.mdx:12]: I think so... it might be something Marty did in Figma.

**Pull request merged**